Clean build - Maven dependency issue

Without an existing Maven cache in ~/.m2, a clean build fails with this error:

```
[ERROR] Failed to execute goal on project app-server:
  Could not resolve dependencies for project com.thoughtworks.go:app-server:jar:1.0:
  The following artifacts could not be resolved: 
  org.openrdf:sesame-onejar:jar:2.3.1, org.springframework:web.servlet:jar:3.1.3.RELEASE:
  Could not find artifact org.openrdf:sesame-onejar:jar:2.3.1
  in local.maven.repo (file:/home/gocd/app-server/local-maven-repo/) -> [Help 1]
```

--- a/app-server/pom.xml
+++ b/app-server/pom.xml
@@ -31,6 +31,10 @@
         <relativePath>../</relativePath>
     </parent>
 
+    <properties>
+      <main.dir>${project.basedir}/..</main.dir>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
 
     <dependencies>
         <dependency>

--- a/jetty6/pom.xml
+++ b/jetty6/pom.xml
@@ -33,6 +33,8 @@
 
     <properties>
         <jetty.version>6.1.23</jetty.version>
+        <main.dir>${project.basedir}/..</main.dir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencies>

--- a/jetty9/pom.xml
+++ b/jetty9/pom.xml
@@ -33,6 +33,8 @@
 
     <properties>
         <jetty.version>9.2.3.v20140905</jetty.version>
+        <main.dir>${project.basedir}/..</main.dir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencies>

